demo/api.md

```diff
@@ -802,11 +798,9 @@ In the following example, please see how pricing content is threaded into the el
 
 #### Popover Slot
 
-Each day within the calendar has a slot that allows a user to add content into an `auro-popover` that will appear when hovering or adding focus to a calendar cell. The user will add their slot content withtin the `auro-datepicker`, with the slot name needing to be in the format of `popover_{month}_{day}_{fullYear}`. 
+Custom content can be added to any day's `auro-popover` in the calendar using a `slot` named following the pattern `date_{MM}_{DD}_{YYYY}` (e.g. `date_01_08_2024`).
 
-**Note**: When the month is a single digit, a zero will need to be added to the front of the month. For example, January would be `01` and not `1`.
-
-In the following example, please see how popover content is threaded into the element's `<slot>`.
+The default use for the popover slot content is to be used in conjunction with a *disabled* `auro-calendar-cell`.
```

Comment:
The popover slot is intended for use with calendar dates that are `disabled` (e.g. before minimum date or after maximum date).